config/services/servicesConfig.go

```diff
@@ -60,6 +61,14 @@ type Instagram struct {
 	AccessToken string
 }
 
+// Mastodon holds the functional configuration settings related to the Mastodon shortcodes
+type Mastodon struct {
+	// The Simple variant of Mastodon is decorated with a basic set of inline styles.
```

Comment:
Is the inline CSS really necessary? And if so, wouldn't it be easier to override the template instead of having a config option?

Reply:
Again here, I just follow the same pattern as other services like Twitter or Instagram.